tools/depends/target/rapidjson/Makefile

```diff
@@ -36,7 +36,7 @@ $(TARBALLS_LOCATION)/$(ARCHIVE):
 $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
 ifeq ($(PREFIX),)
 	@echo
-	@echo "ERROR: please set PREFIX to the kodi install path e.g. make PREFIX=/usr/local"
+	@echo "ERROR: please set PREFIX to the kodi install path e.g. $(MAKE) PREFIX=/usr/local"
```

Comment:
not here please

Reply:
okay
added it here to output make or gmake, depending on which one used

Reply:
it that works, it's perfectly fine, just wasn't 100% sure it works as expected in echo lines

Reply:
tested on FreeBSD and Linux and there it worked